setup/setup_all.sh

```diff
@@ -22,31 +22,32 @@ setup_env_vars() {
   export PROJECT_ID="solutions-template-sandbox"
   export ADMIN_EMAIL="your_email@example.com"
   export REGION="us-central1"
-  export TF_VAR_project_id=$PROJECT_ID
-  export TF_VAR_api_domain=$API_DOMAIN
-  export TF_VAR_web_app_domain=$API_DOMAIN
-  export TF_VAR_admin_email=$ADMIN_EMAIL
+  export API_DOMAIN="localhost"
```

Comment:
The idea of API_DOMAIN is to allow users to override using environment vars without looking to the script here, as in https://github.com/GoogleCloudPlatform/solutions-template#setting-up-google-cloud-project.

This will put API_DOMAIN as hardcoded to localhost for all cases.

Reply:
This file is just a sample file (not used in cookiecutter)

Reply:
This file is used in the cookecutter, because you'll use the bash build_tools/build_templates.sh to update everything in cookiecutter: https://github.com/GoogleCloudPlatform/solutions-template/blob/main/CONTRIBUTING.md#build-cookiecutter-template-folder